x-pack/test/observability_functional/apps/observability/alerts/workflow_status.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+
+  describe('workflow status', function () {
```

Comment:
In Kibana's functional tests the test cases don't stand on their own, but are expected to be run with their whole suite. So this suite represents a user workflow which alternates manipulating an alert and checking the table. Since there is no other way to see the workflow status in the table except by filtering it seems natural to test them in tandem. What different structure would you propose?

Reply:
Can we defer this until after the test authoring activity has reduced a bit? Then we can reorganize the hierarchy to make sure "observability alerts" is in the name of all test cases.